release.py

```diff
@@ -149,6 +155,7 @@ def main():
     parser.add_argument('--allow_dirty', default=False, action='store_true')
     parser.add_argument('--skip_version_bump', default=False, action='store_true')
     parser.add_argument('--skip_latest_tag', default=False, action='store_true')
+    parser.add_argument('--github_token')
```

Comment:
nit: use an env var for this instead, e.g. `os.environ.get('GITHUB_TOKEN')`

Reply:
Done